package.json

```diff
@@ -20,11 +20,11 @@
     },
     "dependencies": {
         "16": "^0.0.2",
-        "@companieshouse/api-sdk-node": "^2.0.113",
+        "@companieshouse/api-sdk-node": "^2.0.147",
         "@companieshouse/node-session-handler": "^4.1.9",
         "@companieshouse/structured-logging-node": "^1.0.8",
```

Comment:
can we update `node-session-handler` and `structured-logging-node` to the latest versions please?

Reply:
its not required for this update but that can be done no problem